tauri-updater/README.md

```diff
@@ -0,0 +1,275 @@
+# Tauri Updater
+---
+> ⚠️ This project is a working project. Expect breaking changes.
+---
+
+The updater is focused on making Tauri's application updates **as safe and transparent as updates to a website**.
+
+Instead of publishing a feed of versions from which your app must select, Tauri updates to the version your server tells it to. This allows you to intelligently update your clients based on the request you give to Tauri.
+
+The server can remotely drive behaviors like rolling back or phased rollouts.
+
+The update JSON Tauri requests should be dynamically generated based on criteria in the request, and whether an update is required.
+
+Tauri's installer is also designed to be fault-tolerant, and ensure that any updates installed are valid and safe.
+
+# Configuration
+
+Once you have your Tauri project ready, you need to configure the updater.
+
+Add this in tauri.config.js
+```json
+"updater": {
+    "active": true,
+    "endpoints": [
+        "https://releases.myapp.com/{target}}/{current_version}}"
+    ],
+    "dialog": true,
+    "pubkey": "",
```

Comment:
```suggestion
    "pubkey": ""
```

Since the Tauri config file is now JSON, having a comma after the last property is not allowed